bridges/ARDMediathekBridge.php

```diff
@@ -4,32 +4,87 @@ class ARDMediathekBridge extends BridgeAbstract {
 	const URI = 'https://www.ardmediathek.de';
 	const DESCRIPTION = 'Feed of any series in the ARD-Mediathek, specified by its path';
 	const MAINTAINER = 'yue-dongchen';
```

Comment:
90% of bridge code has been rewritten.
If you, @Mar-Koeh, are going to maintain this bridge, you can add your github name via comma.

Bridge maintainers are pinged in issues, when error happens.

Also question for @yue-dongchen, are still maintaining this bridge?

Reply:
I didn't know of the API before. I'll stay there and watch out for new issues.

Reply:
> 90% of bridge code has been rewritten. If you, @Mar-Koeh, are going to maintain this bridge, you can add your github name via comma.

if @yue-dongchen is fine with it, I'd join maintaining this bridge.